### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#24] [General]
# 乱翻书第1期 - hello

封面来源，song，我的女朋友，今天是我们在一起的300天

https://kafka3-github-io.vercel.app/posts/1-hello

**Comment by Kafka3** [comment]
11月11号才是三百天🧎‍♂️🧎‍♂️

**Reply by Song-D1** [reply]
随机抽查：30s说出今天是meet___天ʚෆɞ₊

**Reply by Kafka3** [reply]
1382天❤️
